fahrschein/src/main/java/org/zalando/fahrschein/DefaultObjectMapper.java

```diff
@@ -12,13 +12,18 @@
     static final ObjectMapper INSTANCE;
     static {
         final ObjectMapper objectMapper = new ObjectMapper();
-        objectMapper.setPropertyNamingStrategy(PropertyNamingStrategy.SNAKE_CASE);
+        objectMapper.setPropertyNamingStrategy(snake_case());
         objectMapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
         objectMapper.registerModules(new Jdk8Module(), new ParameterNamesModule(), new JavaTimeModule());
         objectMapper.disable(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES);
         INSTANCE = objectMapper;
     }
 
+    @SuppressWarnings("deprecation")
+    private static PropertyNamingStrategy snake_case() {
```

Comment:
Maybe leave a comment here, why you intentionally decided to use a deprecated value? Otherwise, as an innocent reader of the code I would ask myself.